app/routes/kvittering.tsx

```diff
@@ -0,0 +1,20 @@
+import HovedInnhold from '~/komponenter/hovedInnhold/HovedInnhold';
+import { useTekster } from '~/hooks/contextHooks';
+import TekstBlokk from '~/komponenter/tekstblokk/TekstBlokk';
+import { ESanityMappe } from '~/typer/felles';
+import { ETypografiTyper } from '~/typer/typografi';
+import StegIndikator from '~/komponenter/stegindikator/StegIndikator';
+
+export default function Kvittering() {
+  const tekster = useTekster(ESanityMappe.KVITTERING);
+
+  return (
+    <HovedInnhold>
+      <StegIndikator nåværendeSteg={3} />
+      <TekstBlokk
```

Comment:
Her burde vi trekt ut `ANTALL_STEG` i en egen fil i konstater mappen. Den finner man som en konstant i steg indikator. Dette ble vurdert da det ble laget. Da vil man alltid vise at man er på siste steg i kvitterings siden, uten å måtte endre kode her seinare. 

Da kan man sette inn: 

``` javascript
<StegIndikator nåværendeSteg={ANTALL_STEG + 1} />
```

Reply:
Hvorfor trenger man å plusse på 1 dersom man gjør det på denne måten? Hvis man lager en `ANTALL_STEG` burde denne reflektere hvor mange steg som faktisk finnes (3 for oss nå)

Reply:
Ligger et [trellokort](https://trello.com/c/LKsGcPJn/108-fikse-logikk-i-stegindikator) ute for å se mer på logikken for stegindikator, og hvorvidt den kan forenkles! Gjorde ikke noe med dette i denne oppgaven ettersom det ikke egentlig hører til her